mage_ai/frontend/components/RuntimeVariables/index.style.tsx

```diff
@@ -1,28 +1,12 @@
 import styled from 'styled-components';
 
 import dark from '@oracle/styles/themes/dark';
-import { BORDER_RADIUS } from '@oracle/styles/units/borders';
-import { PADDING, UNIT } from '@oracle/styles/units/spacing';
-import { ScrollbarStyledCss } from '@oracle/styles/scrollbars';
-
-export const ContainerStyle = styled.div`
-  border-bottom: 1px solid ${dark.borders.medium};
-  padding: ${PADDING}px;
-`;
-
-export const CardsStyle = styled.div`
-  ${ScrollbarStyledCss}
-
-  height: 80px;
+import { PADDING } from '@oracle/styles/units/spacing';
 
+export const ContainerStyle = styled.div<{ $height: number }>`
```

Comment:
@anniexcheng jw is there any significance behind the `$` in front of the variable name? Versus just using `height`? In terms of consistency with the rest of our prop names, I don't think we name any other FE variables with the `$` in front.

Reply:
Yea, the difference is that with `$height`, `height` won't show up in the DOM but without it, it will! Here's the longer explanation from the [docs](https://styled-components.com/docs/basics#passed-props).

`$height`
<img width="351" alt="image" src="https://github.com/mage-ai/mage-ai/assets/8130751/fc7642c3-8882-4d32-8b11-6fe1c1478670">

`height`
<img width="346" alt="image" src="https://github.com/mage-ai/mage-ai/assets/8130751/9098d147-9024-4040-aaa2-abf1540808e8">

I'm down with either approach though! If we want consistency, I can switch to `height`.

Reply:
Ah, is there a situation where it'd not be ideal to show `height` in the DOM? If it is innocuous, we can remove the `$` for consistency, but good to know.